sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -55,6 +55,15 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+# dest is initialized to a different name in parser.add_argument.
+# This information is needed to maintain consistent behavior
+# between flags and kwargs.
+DEPENDENT_ARGS = {
```

Comment:
Maybe rename `DEPENDENT_ARGS` so that it's clear that it's a mapping? Like `CMD_LINE_FLAG_TO_KWARGS_MAP`.

Or at least make the above comments super clear where the keys/values are coming from. i.e.`no_use_public_ips` is passed when using command line and `use_public_ips` when from dict.

Reply:
Yes. I will add the comments when i do refactoring.